torch/onnx/utils.py

```diff
@@ -729,6 +729,17 @@ def const_if_tensor(arg):
 # help the target of an ONNX export export more efficiently.  However,
 # ONNX doesn't currently formalize inplace. Fortunately, it's sound to drop
 # inplace annotations, but we are losing information this way.
+def _find_symbolic_fn(ns, domain, op_name, opset_version, operator_export_type, msg=''):
+    import torch.onnx.symbolic_registry as sym_registry
+    if not sym_registry.is_registered_op(op_name, domain, opset_version):
+        if operator_export_type == OperatorExportTypes.ONNX_ATEN_FALLTHROUGH:
+            # Use the original node directly
+            return None
+        else:
+            warnings.warn("ONNX export failed on operator {}::{} because "
```

Comment:
I'm wondering if we should keep this as warning. What happens for each different operator export type, if code reach this place?

Reply:
Yes, this warning just adds more info to the error. But it's duplicate. We can remove this.